conans/test/utils/cpp_test_files.py

```diff
@@ -360,6 +362,13 @@ def cpp_hello_conan_files(name="Hello", version="0.1", deps=None, language=0, st
     for i in range(use_additional_infos):
         res += info_tmp % ("EnvVar%d" % i, "UserVar%d" % i)
 
+    scm_placeholder = ""
+    if scm_dict:
+        scm_dict_as_text = ', '.join(["'{}': '{}'".format(k, v) for k,v in scm_dict.items()])
```

Comment:
why a bare ``str(scm_dict)`` is not enough?

Reply:
uhm.... sometimes you need to go the hard way just because... I'll change it :S